centos_install_requirements.sh

```diff
@@ -46,17 +46,6 @@ sudo yum -y install \
   python3-pip \
   wget
 
-# Install tripleo-repos, used to get a recent version of python-virtualbmc
-sudo dnf -y --repofrompath="current-tripleo,https://trunk.rdoproject.org/${DISTRO}-master/current-tripleo" install "python*-tripleo-repos" --nogpgcheck
-sudo tripleo-repos current-tripleo
-
-
-# There are some packages which are newer in the tripleo repos
-# FIXME(stbenjam): On CentOS 7, the version of oniguruma conflicts with
-# the version shipped in the tripleo repos. This needs further
-# investigation.
-sudo yum -y update --exclude=oniguruma
-
```

Comment:
I'm not entirely sure what to do with existing deployments that have RDO repos enabled - at first glance it seems like we'd disable/remove them - but as mentioned here there are actually $many packages that get pulled in from the RDO deps repo, either due to dependencies or because they are newer.

Reply:
Perhaps we could print a message if we detect the RDO repos are installed that they are no longer required by Metal3, and users can disable them if desired? I don't think I'd want to disable the repos in case the user is using them for something else, but just leaving them enabled without saying anything isn't great either.

Reply:
My main concern is more dependency conflicts like the oniguruma one - if we leave the repos around that may happen again, but we can't really fully remove the dependency on the repo on an existing system (on my test machine over 2000 packages are pulled in from the openstack repos!!)

I guess the best we can do is log a warning if we find them, and be sure to not enable them for net-new platforms (e.g CentOS8), then over time we can fully break this dependency